frontend/src/lib/lemon-ui/LemonButton/LemonButton.tsx

```diff
@@ -26,7 +26,7 @@ export interface LemonButtonPropsBase
         | 'role'
         | 'aria-haspopup'
     > {
-    children?: React.ReactNode
+    children?: React.ReactNode | undefined
```

Comment:
Not sure why this is necessary?

Reply:
Any of the changes came from linting errors when running https://github.com/PostHog/posthog/actions/runs/6392211358/job/17349015734. Can't seem to find it now so just removed it, perhaps it was just a local issue.